frontend/src/components/smart/PvPArena.vue

```diff
@@ -372,10 +437,22 @@ export default {
     this.loading = false;
   },
 
+  async updated() {
+    await this.updateSecondsBeforeSeason();
+  },
+
+  destroyed() {
+    if (this.restartEventSubscription.id) {
```

Comment:
Can restartEventSubscription be null here? It will throw an exception if it's possible